components/schema-migrator/migrations/director/20230314160000_add_tags_labels_and_documentation_labels.up.sql

```diff
@@ -0,0 +1,250 @@
+CREATE OR REPLACE VIEW tenants_specifications
+            (tenant_id, id, api_def_id, event_def_id, spec_data, api_spec_format, api_spec_type,
+             event_spec_format, event_spec_type, custom_type, created_at)
+AS
+SELECT DISTINCT t_api_event_def.tenant_id,
+                spec.id,
+                spec.api_def_id,
+                spec.event_def_id,
+                spec.spec_data,
+                spec.api_spec_format,
+                spec.api_spec_type,
+                spec.event_spec_format,
+                spec.event_spec_type,
+                spec.custom_type,
+                spec.created_at
+FROM specifications spec
+         JOIN (SELECT a.id,
+                      a.tenant_id
+               FROM tenants_apis a
+               UNION ALL
+               SELECT e.id,
+                      e.tenant_id
+               FROM tenants_events e) t_api_event_def
+              ON spec.api_def_id = t_api_event_def.id OR spec.event_def_id = t_api_event_def.id;
+
+
+CREATE OR REPLACE VIEW tenants_apps
+            (tenant_id, formation_id, id, name, description, status_condition, status_timestamp, healthcheck_url,
+             integration_system_id, provider_name, base_url, labels, ready, created_at, updated_at, deleted_at, error,
+             app_template_id, correlation_ids, system_number, application_namespace, product_type)
+AS
+SELECT DISTINCT t_apps.tenant_id,
+                t_apps.formation_id,
+                apps.id,
+                apps.name,
+                apps.description,
+                apps.status_condition,
+                apps.status_timestamp,
+                apps.healthcheck_url,
+                apps.integration_system_id,
+                apps.provider_name,
+                apps.base_url,
+                apps.labels,
+                apps.ready,
+                apps.created_at,
+                apps.updated_at,
+                apps.deleted_at,
+                apps.error,
+                apps.app_template_id,
+                apps.correlation_ids,
+                apps.system_number,
+                coalesce(apps.application_namespace, tmpl.application_namespace),
+                tmpl.name AS product_type
+FROM applications apps
+         LEFT JOIN app_templates tmpl ON apps.app_template_id = tmpl.id
+         JOIN (SELECT a1.id,
+                      a1.tenant_id AS tenant_id,
+                      'aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa' AS formation_id
+               FROM tenant_applications a1
+               UNION ALL
+               SELECT apps_subaccounts.id,
+                      apps_subaccounts.tenant_id,
+                      apps_subaccounts.formation_id
+               FROM apps_subaccounts
+               UNION ALL
+               SELECT apps_subaccounts.id,
+                      apps_subaccounts.tenant_id,
+                      'aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa' AS formation_id
+               FROM apps_subaccounts) t_apps
+              ON apps.id = t_apps.id;
+
+CREATE
```

Comment:
minor cosmetic suggestion for consistency purposes: could you inline it `CREATE OR REPLACE VIEW <.....>` instead of `CREATE` being on a separate line

Same applies for the down migration as well.